src/Effects/Effects.Android/Effects/Label/LabelCustomFont.cs

```diff
@@ -18,7 +18,7 @@ protected override void OnAttached()
                 return;
             else
             {
-                var effect = (FormsCommunityToolkit.Effects.CustomFontEffect)Element.Effects.FirstOrDefault(item => item is FormsCommunityToolkit.Effects.CustomFontEffect);
+                var effect = (FormsCommunityToolkit.Effects.LabelCustomFont)Element.Effects.FirstOrDefault(item => item is FormsCommunityToolkit.Effects.LabelCustomFont);
```

Comment:
Hmm the class itself has the same name! So full name needed to correctly identify...